### ISSUE TITLE
push does not support advanced tags

### ISSUE THREAD

**Issue by shurrman** [aws/amazon-ecs-cli#256]
Hi! I tried to tag same image with different tags - stuff like dashes/underscores is not supported:

```
# docker tag 50e69f949efc surge:alexandraalexandrov
# ecs-cli push surge:alexandraalexandrov
INFO[0000] Getting AWS account ID...
INFO[0000] Tagging image                                 image="surge:alexandraalexandrov" repository="xxxxxxxxxxxxxxx.dkr.ecr.eu-west-1.amazonaws.com/surge" tag=alexandraalexandrov
INFO[0000] Image tagged
INFO[0000] Pushing image                                 repository="xxxxxxxxxxxxxxxx.dkr.ecr.eu-west-1.amazonaws.com/surge" tag=alexandraalexandrov
INFO[0001] Image pushed

```
Now
```
# docker tag 50e69f949efc surge:alexandraalexandrov-2
# ecs-cli push surge:alexandraalexandrov-2
ERRO[0000] Error executing 'push': Please specify the image name in the correct format [ECR_REPOSITORY[:TAG]]
```

When I use `aws ecr get-login` and then docker push - no problem with such tags.

**Comment by yinshiua** [comment]
Are you still seeing the same error? I wasn't able to reproduce this. 
```
$ docker tag 00f017a8c2a6 busybox:alexandraalexandrov-2
$  ecs-cli push busybox:alexandraalexandrov-2           
INFO[0000] Getting AWS account ID...                    
INFO[0000] Tagging image                                 image="busybox:alexandraalexandrov-2" repository="*************.dkr.ecr.ca-central-1.amazonaws.com/busybox" tag=alexandraalexandrov-2
INFO[0000] Image tagged                                 
INFO[0000] Pushing image                                 repository="="*************..dkr.ecr.ca-central-1.amazonaws.com/busybox" tag=alexandraalexandrov-2
INFO[0002] Image pushed    
```

**Comment by yinshiua** [comment]
@shurrman Sorry for the confusion, this was a bug on our end and we already had a fix. PR https://github.com/aws/amazon-ecs-cli/pull/240

**Comment by yinshiua** [comment]
Released in version 0.6.0
